omod/src/main/java/org/openmrs/module/shr/rest/web/controller/EncounterController.java

```diff
@@ -77,7 +61,12 @@ public Object postDocument(
 			@RequestParam(value = "providerId", required = true) String providerId,
 			@RequestParam(value = "providerIdType", required = true) String providerIdType,
 			@RequestParam(value = "encounterType", required = true) String encounterType,
-			@RequestParam(value = "formatCode", required = true) String formatCode,
+			@RequestParam(value = "typeCode_code", required = true) String typeCode_code,
+			@RequestParam(value = "typeCode_codingScheme", required = true) String typeCode_codingScheme,
```

Comment:
Stick to the camelCase notation for these parameter names, especially in Java code